test/integration/tasks.test.js

```diff
@@ -313,6 +313,19 @@ describe("Tasks", function () {
       percentCompleted: 50,
     };
 
+    const taskData = {
+      title: "Test task",
+      type: "feature",
+      endsOn: 1234,
+      startedOn: 4567,
+      status: "VERIFIED",
+      percentCompleted: 10,
+      participants: [],
+      completionAward: { [DINERO]: 3, [NEELAM]: 300 },
+      lossRate: { [DINERO]: 1 },
+      isNoteworthy: true,
+    };
```

Comment:
This is being re-used a lot create a function in fixtures and call it with valid statuses